pyproject.toml

```diff
@@ -37,6 +39,12 @@ line_length = 79
 python_version = 3.11
 strict = true
 
+[tool.django-stubs]
+django_settings_module = "task_manager.settings"
+
+[tool.poetry.dev-dependencies]
+django-stubs = {extras = ["compatible-mypy"], version = "^5.0.2"}
+
```

Comment:
should be removed as you have the same stubs listed in `[tool.poetry.group.dev.dependencies]` section